tests/integration/update_cluster/ha_gce/kubernetes.tf

```
@@ -564,7 +564,7 @@ resource "google_compute_instance_template" "master-us-test1-a-ha-gce-example-co
     preemptible         = false
   }
   service_account {
-    email  = "default"
+    email  = "control-plane-ha-gce-example-com@testproject.iam.gserviceaccount.com"
```

when we get to the specific PR for this change, I believe this should be a terraform link rather than a literal

```suggestion
    email  = google_service_account.serviceaccount-control-plane.email
```

The TerraformLink code looks correct:

https://github.com/kubernetes/kops/blob/d48e7ab52e00c90d0a3564a81d500dc061f6fa2f/upup/pkg/fi/cloudup/gcetasks/serviceaccount.go#L213-L226

which makes me wonder if we're setting the SA task's `Shared` field incorrectly... 🤔

---

Ah, you're quite right - I see you did this in #12995 - thank you!